src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -103,7 +112,7 @@ const UserProfileModal: React.FunctionComponent<PropTypes> = ({
                     {/*
                     // @debt typing - need to support known User interface with unknown question keys
                     // @ts-ignore */}
-                    {fullUserProfile[question.name] || //@debt typing - look at the changelog, was this a bug?
+                    {userProfile[question.name] || //@debt typing - look at the changelog, was this a bug?
```

Comment:
Maybe?

```suggestion
                    {userProfile[question.name] ?? //@debt typing - look at the changelog, was this a bug?
```

Is this @debt still an issue?

Reply:
Yes, it is.